smart-agent/src/test/java/org.smartdata.agent/TestSmartAgent.java

```diff
@@ -30,10 +31,14 @@
 import org.smartdata.server.engine.cmdlet.agent.messages.AgentToMaster.RegisterNewAgent;
 import org.smartdata.server.engine.cmdlet.agent.messages.MasterToAgent;
 
+import java.security.Permission;
+
 public class TestSmartAgent extends ActorSystemHarness {
```

Comment:
Let's use our common approach <ClassName>Test

Reply:
I think we can we can keep this name for uniformity, because all other tests in the project also start with `Test*`